Description: Disable features that require EFI110
 gnu-efi doesn't support EFI 1.10, so disable anything that requires it.
Author: Julien BLACHE <jblache@debian.org>

Index: refit-0.14/refit/main.c
===================================================================
--- refit-0.14.orig/refit/main.c	2010-04-19 18:38:11.930932047 +0200
+++ refit-0.14/refit/main.c	2010-04-19 18:37:59.327931990 +0200
@@ -925,6 +925,8 @@
     }
 }
 
+
+#ifdef DEBIAN_ENABLE_EFI110
 //
 // pre-boot driver functions
 //
@@ -1036,6 +1038,7 @@
     // connect all devices
     ConnectAllDriversToAllControllers();
 }
+#endif /* DEBIAN_ENABLE_EFI110 */
 
 //
 // main entry point
@@ -1072,7 +1075,9 @@
     
     // further bootstrap (now with config available)
     SetupScreen();
+#ifdef DEBIAN_ENABLE_EFI110
     LoadDrivers();
+#endif /* DEBIAN_ENABLE_EFI110 */
     ScanVolumes();
     DebugPause();
     
